### PR TITLE
changed uncompleted to incompleted

### PR DESCRIPTION
**Pull request by torbenbru**
Changed every uncompleted to incompleted.
#775

### CODE REVIEW

File: classes/report/fields/runner/tests/uncompleted/cli.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?php
 
-namespace mageekguy\atoum\report\fields\runner\tests\uncompleted;
+namespace mageekguy\atoum\report\fields\runner\tests\incompleted;
```
Comment (Grummfy) [inline classes/report/fields/runner/tests/uncompleted/cli.php]: the path of the file should also be changed

File: tests/units/classes/report/fields/runner/tests/uncompleted/cli.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace mageekguy\atoum\tests\units\report\fields\runner\tests\uncompleted;
+namespace mageekguy\atoum\tests\units\report\fields\runner\tests\incompleted;
```
Comment (Grummfy) [inline tests/units/classes/report/fields/runner/tests/uncompleted/cli.php]: we need to change the path to the file too

**Comment by Grummfy** [comment]
@Hywan @jubianchi this perhaps break some internal api, so it can have some impact on existing extension

File: classes/report/fields/runner/tests/uncompleted/cli.php (hunk 1 above)
Comment (torbenbru) [reply classes/report/fields/runner/tests/uncompleted/cli.php]: oh yeah overlooked that one

**Comment by torbenbru** [comment]
well, if uncompleted is valid, this pull request definitely doesn't need to be merged. Waiting for opinions before closing.

**Pull request closed without merging**